testsuite/tests/tool-ocamldoc-html/Makefile

```diff
@@ -33,10 +33,11 @@ default:
 	fi
 
 .PHONY: run
-run: *.mli
-	@for file in *.mli; do \
+run: *.mli *.ml
+	@for file in *.ml *.mli; do \
 	  printf " ... testing '$$file'"; \
 	  F="`basename $$file .mli`"; \
+	  F="`basename $$F .ml`"; \
```

Comment:
The mli are still tested: I am updating `F`  by removing `.ml` extensions after having removed the `.mli` ones: as far as I can see on its man page, `basename` can only removes one suffix at a time.